### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by hedudelgado**
Add the commands to update Angular and the mno-enterprise gem .

### CODE REVIEW

**Comment by hedudelgado** [comment]
@x4d3

**Comment by alexnoox** [comment]
Hi guys.
Thanks for the doc update, I just want to clarify that:
- The rake command for angular update mnoe-angular to the version in the bower.json.
- The bundle command update the version of the gem according to the gemfile. 
They do not update to the last version (master).
As stated with oli, tags should be used to fetch version of mnoe, mnoe-angular version to use being defined in the bower.json of the gem.
Cheers.

File: README.md [hunk 1]
```diff
@@ -390,6 +390,17 @@ The v3 is currently being finalised. There are some minor configuration options
 
 If deploying to UAT, the Impac! URLs need to be manually replaced. Search the "/public" directory for "http://localhost:4000" and replace by "https://api-impac-uat.maestrano.io". Save the files and deploy.
 
+## Update de front end to the newest version
+```
+Rake mnoe:frontend:update
```
Comment (x4d3) [inline README.md]: it should be lowercase. rake

File: README.md [hunk 2]
```diff
@@ -390,6 +390,17 @@ The v3 is currently being finalised. There are some minor configuration options
...
+```
+
+## Update the enterprise to the newest version
+```
+Bundle update mno-enterprise
```
Comment (x4d3) [inline README.md]: it should be lowercase.

**Comment by hedudelgado** [comment]
@alexnoox  sorted. =)

**Pull request merged**